packages/block-library/src/search/edit.native.js

```diff
@@ -14,37 +14,39 @@ import {
 	InspectorControls,
 } from '@wordpress/block-editor';
 import {
-	Button,
 	PanelBody,
 	SelectControl,
 	ToggleControl,
+	Icon,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { search } from '@wordpress/icons';
 import { useRef, useEffect, useState } from '@wordpress/element';
+import { withPreferredColorScheme } from '@wordpress/compose';
```

Comment:
Since this is a functional component what do you think if we use `usePreferredColorSchemeStyle` to avoid using a HOC to get the current color scheme?

Reply:
Sounds reasonable to me. I've implemented this in f1a9b8fb52a0736349b8bafd252efbf688608c3d